src/scrape/get_asteroid_30x30.py

```diff
@@ -0,0 +1,81 @@
+from tqdm import tqdm
+import pandas as pd
+import time
+import os
+from argparse import ArgumentParser
+import subprocess
+from utils import (
+    save_progress,
+    load_progress,
+    get_mover_data,
+)
+from config import (
+    BASE,
+    IMAGE_FOLDER,
+    CSV_FOLDER,
+    META_DATA_COLUM_NAMES,
+    POSITION_TABLE_COLUMN_NAMES,
+)
+
+image_table_labels = META_DATA_COLUM_NAMES[1:]
+position_table_labels = ["pos_" + label for label in POSITION_TABLE_COLUMN_NAMES[1:]]
+bad_request_output = "<p>\n nice try.\n <br/>\n logged.\n <br/>\n bye.\n</p>\n"
+
+if __name__ == "__main__":
+    parse = ArgumentParser()
+    parse.add_argument("--mover_file", type=str, default="movers.csv")
+    parse.add_argument("--progress_file", type=str, default="scraper_progress.txt")
+
+    args = parse.parse_args()
+
+    mover_totas_csv = pd.read_csv(f"{CSV_FOLDER}/{args.mover_file}")
+    tracking_file = args.progress_file
+
+    upload_bucket = "gs://mlp-asteroid-data/csv/"
+    upload_checker = 0
+
+    mover_image_csv_file = args.mover_file.split("/")[-1]
+    meta_data_csv = f"{mover_image_csv_file.split('.')[0]}_image_meta_data.csv"
+    meta_data_csv_path = f"{CSV_FOLDER}/{meta_data_csv}"
+    if not os.path.exists(meta_data_csv_path):
```

Comment:
A lot of this could probably go to separate functions, but fine with me for now.